lib/diagnostic.js

```diff
@@ -5,22 +5,36 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+  let result = arguments[0];
+  for (let i=1;i<arguments.length;i++){
+    if (result > arguments[i]){
+      result = arguments[i];
+    }
+  }
+  return result
 };
```

Comment:
Pay attention to your linter and/or run `grunt nag` from the console to catch typos/omitted semi colons!